tests/components/tessie/test_sensor.py

```diff
@@ -0,0 +1,38 @@
+"""Tessie sensor platform tests."""
+from homeassistant.components.sensor import DOMAIN as SENSOR_DOMAIN
+from homeassistant.const import STATE_UNKNOWN
+from homeassistant.core import HomeAssistant
+
+from .common import DOMAIN, TEST_VEHICLES, URL_VEHICLES, setup_platform
+
+from tests.common import load_json_object_fixture
+from tests.test_util.aiohttp import AiohttpClientMocker
+
+STATES = load_json_object_fixture("vehicles.json", DOMAIN)["results"][0]["last_state"]
+
+
+async def test_sensors(
+    hass: HomeAssistant, aioclient_mock: AiohttpClientMocker
+) -> None:
+    """Tests that the sensors are correct."""
+
+    aioclient_mock.get(
+        URL_VEHICLES,
+        text=TEST_VEHICLES,
+    )
+
+    await setup_platform(
+        hass,
+        [SENSOR_DOMAIN],
```

Comment:
From what I have been told in the past, you should just always set up all platforms, just to make sure one platform doesn't cause an issue with another.